python/src/NonLinearLeastSquaresCalibration_doc.i.in

```diff
@@ -0,0 +1,72 @@
+%feature("docstring") OT::LinearLeastSquaresCalibration
```

Comment:
this docstring seems to refer to LinearLeastSquaresCalibration

Reply:
done, sorry for the copy/paste error